azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -701,6 +728,8 @@ func resourceArmVirtualMachineScaleSetRead(d *schema.ResourceData, meta interfac
 		return fmt.Errorf("[DEBUG] Error setting Virtual Machine Scale Set Sku error: %#v", err)
 	}
 
+	d.Set("identity", flattenAzureRmVirtualMachineScaleSetIdentity(resp.Identity))
```

Comment:
can we change this to:

```
if err := d.Set("identity", flattenAzureRmVirtualMachineScaleSetIdentity(resp.Identity)); err != nil {
  return fmt.Error("Error flattening `identity`: %+v", err)
}
```

this allows any bugs (such as the schema not matching) to be caught

Reply:
Done